documentation/modules/con-securing-client-authorization.adoc

```diff
@@ -36,7 +36,9 @@ For more information on configuring super user access to brokers, see xref:con-s
 
 == User quotas
 
-You can configure the `spec` for the `KafkaUser` resource to enforce quotas so that a user does not exceed access to Kafka brokers based on a byte threshold or a time limit of CPU utilization.
+You can configure the `spec` for the `KafkaUser` resource to enforce quotas so that a user does not exceed access to Kafka brokers.
```

Comment:
```suggestion
You can configure the `spec` for the `KafkaUser` resource to enforce quotas so that a user does not exceed a configured level of access to Kafka brokers.
```

Reply:
Not sure if that's accurate. "Exceed access to Kafka brokers" confused me.

Reply:
Better